Task 2/NamishaGoyalPortfolio

```diff
@@ -0,0 +1,456 @@
+<!DOCTYPE html>
```

Comment:
Good work @namigoel!

You may want to do some slight restructuring.

I see that you have a `.html` file. You should add a `.html` extension to your file.

So the final folder structure should be something like:

`Task 2/NamishaGoyalPortfolio/index.html`